--- vmdkutil-1.0.1/vmdkutil.cpp.orig	2007-10-16 10:18:10.000000000 +0200
+++ vmdkutil-1.0.1/vmdkutil.cpp	2022-01-03 06:13:46.342699412 +0100
@@ -45,6 +45,7 @@
 #define _FILE_OFFSET_BITS	64
 #endif
 
+#include <cstring>
 #include <iostream>
 #include <string>
 #include <map>
@@ -64,6 +65,7 @@
 #define OPEN_WFLAGS (O_BINARY | O_RDWR)
 #define OPEN_MODE	(S_IREAD | S_IWRITE)
 #else
+#include <unistd.h>
 #define tell(fd)	lseek(fd, 0, SEEK_CUR)
 #define OPEN_FLAGS	(O_LARGEFILE | O_RDWR | O_TRUNC | O_CREAT)
 #define OPEN_RFLAGS	(O_RDONLY)
